schemas/senv_data.fbs

```diff
@@ -0,0 +1,21 @@
+//NOTE: This schema is under development.
+//If you decide to use it, be aware that backwards compatibility is not guaranteed.
+
+//Used to transmit fast sample environment data
+
+file_identifier "senv";
+
+enum Location : byte { Unknown = 0, Start, Middle, End }
+
+table senv_data {
```

Comment:
We've used CamelCase for tables and snake_case for fields in other schema.
Now we're using LLVM guidelines maybe it makes sense to use CamelCase for fields, but tables should still be CamelCase.